sig-list.md

```diff
@@ -47,7 +47,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
 |[Network](sig-network/README.md)|[network](https://github.com/kubernetes/kubernetes/labels/sig%2Fnetwork)|* [Bowei Du](https://github.com/bowei), Google<br>* [Michael Zappa](https://github.com/mikezappa87), Microsoft<br>* [Shane Utt](https://github.com/shaneutt), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-network)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-network)|* Gateway API Meeting (Americas): [Mondays at 15:00 PT (Pacific Time) (weekly)](https://zoom.us/j/441530404)<br>* Gateway API Meeting (Europe): [Tuesday (starting September 10th 2024)s at 08:00 PT (Pacific Time) (bi-weekly)](https://zoom.us/j/441530404)<br>* Ingate Meeting: [Fridays at 06:00 PT (Pacific Time) (biweekly)](https://zoom.us/j/94625479998)<br>* Network Policy API Meeting: [Tuesdays at 9:00 PT (Pacific Time) (bi-weekly)](https://zoom.us/j/96264742248)<br>* SIG Network InGate Meeting: [Fridays at 6:00 PT (Pacific Time) (monthly)](https://zoom.us/j/94625479998)<br>* SIG Network Ingress NGINX Meeting: [Thursdays at 8:00 PT (Pacific Time) (monthly)](https://zoom.us/j/98377891310)<br>* SIG Network Meeting: [Thursdays at 09:00 PT (Pacific Time) (biweekly)](https://zoom.us/j/361123509)<br>* SIG Network Multi-Network Meeting: [Tuesdays at 06:00 PT (Pacific Time) (biweekly starting Tuesday Oct 28, 2025)](https://zoom.us/j/95680858961?pwd=M1c2TTdMZHpMUUtIYXRpbjRobkNJZz09)<br>
-|[Node](sig-node/README.md)|[node](https://github.com/kubernetes/kubernetes/labels/sig%2Fnode)|* [Sergey Kanzhelev](https://github.com/SergeyKanzhelev), Google<br>* [Peter Hunt](https://github.com/haircommander), Red Hat<br>* [Mrunal Patel](https://github.com/mrunalp), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-node)<br>* [Mailing List](https://groups.google.com/a/kubernetes.io/g/sig-node)|* Main SIG Meeting: [Tuesdays at 10:00 PT (Pacific Time) (weekly)](https://zoom.us/j/4799874685)<br>* Weekly CI/Triage Meeting: [Wednesdays at 10:00 PT (Pacific Time) (weekly)](https://zoom.us/j/93387714284?pwd=NHNpT2xtOGtwNnVQZ1JXdkNlQ1RMQT09)<br>
+|[Node](sig-node/README.md)|[node](https://github.com/kubernetes/kubernetes/labels/sig%2Fnode)|* [Sergey Kanzhelev](https://github.com/SergeyKanzhelev), Google<br>* [Peter Hunt](https://github.com/haircommander), Red Hat<br>* [Mrunal Patel](https://github.com/mrunalp), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-node)<br>* [Mailing List](https://groups.google.com/a/kubernetes.io/g/sig-node)|* Main SIG Meeting: [Tuesdays at 10:00 PT (Pacific Time) (weekly)](https://zoom.us/j/91437143016)<br>* Weekly CI/Triage Meeting: [Wednesdays at 10:00 PT (Pacific Time) (weekly)](https://zoom.us/j/93387714284?pwd=NHNpT2xtOGtwNnVQZ1JXdkNlQ1RMQT09)<br>
```

Comment:
should we apply the same password as all other meetings have:

https://github.com/kubernetes/community/blob/44fb2a7dec12d7525b423f4ce22c335d59edfadf/communication/zoom-guidelines.md?plain=1#L52